centralized-sampling-tests/integration-tests/src/main/java/com.amazon.tests/CentralizedSamplingIntegrationTests.java

```diff
@@ -354,8 +355,8 @@ public static void sampleRulesTests() throws IOException, InterruptedException {
                             break;
                         } else if (k < GenericConstants.MAX_RETRIES - 1) {
                             logger.warn("Test failed here, attempting retry");
+                            TimeUnit.SECONDS.sleep(GenericConstants.RETRY_WAIT);
```

Comment:
Could you elaborate the reason move sleep to here and increase the intervals?

Reply:
>the reason move sleep to here

It doesn't make any difference here, this is more for consistency.

>and increase the intervals?

1 second polling intervals is too frequent, could cause performance impact. The retries should handle this polling interval increase.